docs/docs/data-fetching.md

```diff
@@ -74,6 +74,7 @@ exports.sourceNodes = async ({
   const resultData = await result.json()
   // create node for build time data example in the docs
   createNode({
+    //nameWithOwner and url are arbitrary fields from the data
```

Comment:
```suggestion
    // nameWithOwner and url are arbitrary fields from the data
```

This is nit-picky but I like seeing these comments as consistent. I'll see if I can push to your branch to make an update like this 🙂